gnd/src/main/java/com/google/android/gnd/system/CameraManager.java

```diff
@@ -37,61 +41,63 @@
   /** Used to identify requests coming from this application. */
   static final int CAPTURE_PHOTO_REQUEST_CODE = CameraManager.class.hashCode() & 0xffff;
 
+  private final Context context;
   private final PermissionsManager permissionsManager;
   private final ActivityStreams activityStreams;
 
   @Inject
-  public CameraManager(PermissionsManager permissionsManager, ActivityStreams activityStreams) {
+  public CameraManager(
+      @ApplicationContext Context context,
+      PermissionsManager permissionsManager,
+      ActivityStreams activityStreams) {
+    this.context = context;
     this.permissionsManager = permissionsManager;
     this.activityStreams = activityStreams;
   }
 
   /** Launches the system's photo capture flow, first obtaining permissions if necessary. */
   @Cold
-  public Maybe<Bitmap> capturePhoto() {
+  public Maybe<Nil> capturePhoto(File destFile) {
```

Comment:
Would it make more sense for this and related methods to return `Completable<Boolean>`?